local-volume/helm/README.md

```diff
@@ -44,21 +44,46 @@ The generated specs can be further customized as needed (usually not necessary),
+| common.namespace                       | Namespace where provisioner runs.                                                                     | str      | `default`                                                  |
+| common.useAlphaAPI                     | If running against pre-1.10 k8s version, the `useAlphaAPI` flag must be enabled.                      | bool     | `false`                                                    |
+| common.useJobForCleaning               | Is set to true, provisioner will use jobs-based block cleaning.                                       | bool     | `false`                                                    |
+| common.configMapName                   | Provisioner ConfigMap name.                                                                           | str      | `local-provisioner-config`                                 |
+| classes.[n].name                       | StorageClass name.                                                                                    | str      | `-`                                                        |
+| classes.[n].hostDir                    | Path on the host where local volume of this storage class are mounted.                                | str      | `-`                                                        |
+| classes.[n].mountDir                   | Optionally specify mount path of local volumes. By default, we use same path as hostDir in container. | str      | `-`                                                        |
+| classes.[n].blockCleanerCommand        | List of command and arguments of block cleaner command.                                               | list     | `-`                                                        |
+| classes.[n].storageClass               | Create storage class for this class and configure it optionally.                                      | bool/map | `false`                                                    |
+| classes.[n].storageClass.reclaimPolicy | Specify reclaimPolicy of storage class, available: Delete/Retain.                                     | str      | `Delete`                                                   |
+| daemonset.name                         | Provisioner DaemonSet name.                                                                           | str      | `local-volume-provisioner`                                 |
+| daemonset.image                        | Provisioner image.                                                                                    | str      | `quay.io/external_storage/local-volume-provisioner:v2.1.0` |
+| daemonset.imagePullPolicy              | Provisioner DaemonSet image pull policy.                                                              | str      | `Always`                                                   |
```

Comment:
I'm thinking it may be better to just use the default imagePullPolicy if not set.